### PR TITLE
make ddl execute in parallel at proxy

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: Wei Liu <wei.liu@zilliz.com>
issue: #26970

### CODE REVIEW

File: internal/proxy/task_scheduler.go [hunk 1]
```diff
@@ -480,7 +480,7 @@ func (sched *taskScheduler) definitionLoop() {
 		case <-sched.ddQueue.utChan():
 			if !sched.ddQueue.utEmpty() {
 				t := sched.scheduleDdTask()
-				sched.processTask(t, sched.ddQueue)
+				go sched.processTask(t, sched.ddQueue)
```
Comment (jiaoew1991) [inline internal/proxy/task_scheduler.go:483]: Should use a routine pool be used to control concurrency?

Comment (weiliu1031) [reply internal/proxy/task_scheduler.go:483]: this pr is a quick fix, which will cherry-pick to 2.2 branch. and a new task scheduler which remove the task queue and control the concurrency will be a large change, which will be done in next pr for 2.3 branch only

**Comment by jaime0815** [comment]
DDL should be executed serially？

**Pull request closed without merging**